mapeditor/windownewmap.cpp

```diff
@@ -37,8 +37,10 @@ WindowNewMap::WindowNewMap(QWidget *parent) :
 	show();
 
 	//setup initial parameters
-	mapGenOptions.setWidth(ui->widthTxt->text().toInt());
-	mapGenOptions.setHeight(ui->heightTxt->text().toInt());
+	int width = ui->widthTxt->text().toInt();
+	int height = ui->heightTxt->text().toInt();
+	mapGenOptions.setWidth(width ? width : 1);
```

Comment:
is it kind of default value? Is not 1 too small?

Reply:
There is an assertion that width and size must be bigger 0.
1 is small but I can imagine map something like 1x200 :)